net-vpn/headscale/headscale-0.15.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module systemd
+
+GENTOO_DEPEND_ON_PERL=no
+
+DESCRIPTION="An open source, self-hosted implementation of the Tailscale control server."
+HOMEPAGE="https://github.com/juanfont/headscale"
+SRC_URI="
+	https://github.com/juanfont/headscale/archive/refs/tags/v0.15.0.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~jsmolic/distfiles/${P}-deps.tar.xz
+"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	acct-user/headscale
+	acct-group/headscale
+"
+
+RDEPEND="net-firewall/iptables"
+
+src_compile() {
+	if [[ -f Makefile ]] || [[ -f GNUmakefile ]] || [[ -f makefile ]]; then
+		emake  install-protobuf-plugins && emake build
+	fi
+}
+
+src_test() {
+	emake test
+}
+
+src_install() {
+	insinto /usr/bin/
```

Comment:
`dobin` already install to this location (check https://devmanual.gentoo.org/function-reference/install-functions/index.html)